scripts/setup_env.sh

```diff
@@ -4,4 +4,6 @@
 # shellcheck disable=SC2046
 export $(grep -v '^#' modelci/env-mongodb.env | xargs -d '\r\n')
 
+export $(grep -v '^#' modelci/env-backend.env | xargs -d '\r\n')
```

Comment:
why exports twice？

Reply:
different environment, one for mongodb, one for backend.

Reply:
please make comments

Reply:
done